packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -122,14 +126,6 @@ class TransactionService {
   startTransaction(name, type, options) {
     const perfOptions = this.createPerfOptions(options)
 
-    if (!type) {
```

Comment:
Already taken care in `span-base` https://github.com/elastic/apm-agent-rum-js/blob/0116f68aca3eccf5bc3fafcee6d13e87de8d656b/packages/rum-core/src/performance-monitoring/span-base.js#L32